tests/stdlib/tparsopt.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-disabled: true
+    disabled: true
```

Comment:
Known issue, design of parse options is dumb

Reply:
Do we have a issue number?

Reply:
Don't want to track an issue for this, currently the standard library is mostly out of scope unless required to unblock compiler work. The noise and fielding PRs in that way is more work than it's worth right now.